site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -238,6 +238,101 @@ Apply an Annotation with the prefix "agones.dev/sdk-" to the backing `GameServer
 curl -d '{"key": "foo", "value": "bar"}' -H "Content-Type: application/json" -X PUT http://localhost:${AGONES_SDK_HTTP_PORT}/metadata/annotation
 ```
 
+### Counters
+
+{{< alpha title="Counters" gate="Counters" >}}
```

Comment:
```suggestion
{{< alpha title="Counters and Lists" gate="CountsAndLists" >}}
```

There is only one feature flag for both Counters and Lists - so it needs to be both - hence I left in the top level header of "Counters and Lists" over each "Counters" and "Lists" heading.

Reply:
Included the header.